### ISSUE TITLE
autosave event feature

### ISSUE THREAD

**Issue by jc-anguiano** [sparksuite/simplemde-markdown-editor#627]
currently autosave function don't expose any event

**Comment by kunicki** [comment]
Yes this would be great if there was an autosave event that we could optionally use for persisting changes ourselves. The on change event is useful, but autosave with interval would be better. :-)
